staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -56,6 +56,17 @@ type debugError interface {
 	DebugError() (msg string, args []interface{})
 }
 
+type AdditionalFlags struct {
```

Comment:
The word `Additional` here feels a bit vague. Maybe something like `ApplyFlags` would be better?

Reply:
Done